v2/header.go

```diff
@@ -51,3 +56,52 @@ func XGoogHeader(keyval ...string) string {
 	}
 	return buf.String()[1:]
 }
+
+// version is a package internal global variable for testing purposes.
+var version = runtime.Version
+
+// VersionUnknown is only used when the runtime version cannot be determined.
+const VersionUnknown = "UNKNOWN"
```

Comment:
Does anything use this, if not does should this be exported?

Reply:
Read your PR description -- seems fair. I think this should be GoVersionUnknown though.

Reply:
My thinking was that if we document the default return value to be `"UNKNOWN"`, someone _could_ code against that. So instead of being locked into a static string, let's use a variable so that if we ever needed to change it, it would be transparent to those using `VersionUnknown` instead of the raw string.

I'm not super stuck on the idea though, happy to remove it. wdyt?

Reply:
I am not sure if anyone will ever use it -- at least I have not seen a request. Seems more for our internal tooling than anything else. If users want to know their Go version they should call the stdlib themselves I think and not try to use a value we put in a header.

Reply:
Cool! Fair enough.

Reply:
Done.